src/components/Accordion.js

```diff
@@ -0,0 +1,93 @@
+const menuOpen = "collapse show";
+const buttonOpenAll = "Open All";
+const buttonCloseAll = "Close All";
+
+if (!String.prototype.includes) {
```

Comment:
@tmccoy529 I am going to try and get this into it's own file while you are out.

Reply:
@tmccoy529 take a look at this [change](https://github.com/baltimorecounty/dotgov-javascript/pull/7/commits/261e42d44e5a5cc77fad3dff788ec0d3014fce50). 

The ie issues were actually caused missing (var or let) in the for loop. After that was updated I was able to import the polyfill as expected.

Reply:
oh crap, that's because I had var i = 0 above then took it out and forgot to add var to the loop. Thanks for the catch on that. This is working now.